packages/cli/src/commands/election/vote.test.ts

```diff
@@ -8,7 +8,7 @@ import {
   registerAccountWithLockedGold,
   setupGroupAndAffiliateValidator,
 } from '../../test-utils/chain-setup'
-import { stripAnsiCodes, testLocally } from '../../test-utils/cliUtils'
+import { stripAnsiCodesAndTxHashes, testLocally } from '../../test-utils/cliUtils'
```

Comment:
can you tell me what extra thing this is doing? I mean besides striping txhash or why is that needed / not likely to deceive us?

Reply:
removed it as not longer necessary